### DISCUSSION TITLE
Reference?

### DISCUSSION THREAD

**Discussion by OliveHBLee** [easystats/parameters#909] [General]
Hi,

I used the function on my analysis and I am about to submit my paper. How can I get your reference? 
Olive

**Comment by OliveHBLee** [comment]
I got the reference info. thanks

**Reply by bwiernik** [reply]
For any R package, you can use the citation("performance") function to see the preferred citation. 

For easystats, you can also use easystats::cite_easystats()

**Reply by OliveHBLee** [reply]
Thanks.
